awx/conf/fields.py

```diff
@@ -96,6 +100,17 @@ def to_internal_value(self, data):
         self.fail('type_error', input_type=type(data))
 
 
+class StringListPathField(StringListField):
+
+    def __init__(self, **kwargs):
+        super(StringListPathField, self).__init__(**kwargs)
+        self.validators.append(lambda sp: list(map(validate_system_path, sp)))
```

Comment:
These nested functions are a little hard to parse, would a list comprehension be clearer to the reader?

```python
self.validators.append(lambda paths: [validate_system_path(path) for path in paths])
```

Reply:
Ah yes, this is definitely more readable. I will do that!